pkgs/development/python-modules/pymumble/default.nix

```
@@ -0,0 +1,33 @@
+{ buildPythonPackage,
+  fetchFromGitHub,
+  isPy27,
+  opuslib,
+  protobuf,
+  stdenv,
```

since we're just uring stdenv to access lib, we could just access lib directly
```suggestion
  lib,
```

---

thanks for the advice, never realized its the same `lib`

---

it's there just for convenience